server/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -419,6 +419,8 @@ public void testFindEarliestLastModifiedAge() throws IOException {
     }
 
     public void testStats() throws IOException {
+        assertThat(translog.getCurrent().getLastModifiedTime(), lessThanOrEqualTo(System.currentTimeMillis()));
+        long lastModifiedAge = System.currentTimeMillis() - translog.getCurrent().getLastModifiedTime();
```

Comment:
I think we should eventually fix the time source used by the translog to not be `System.currentTimeMillis()` here and use the `ThreadPool` time suppliers we already use elsewhere. But for now this should be fine :)

Reply:
> I think we should eventually fix the time source used by the translog to not be System.currentTimeMillis() here and use the ThreadPool time suppliers we already use elsewhere

++.